tools/deploy_prow.sh

```diff
@@ -71,6 +73,12 @@ function launchMonitoring(){
   make -C config/prow/monitoring
 }
 
+function launchARC(){
+  kubectl apply -f https://github.com/actions/actions-runner-controller releases/download/v0.22.0/actions-runner-controller.yaml
```

Comment:
Typo in the URL (` ` instead of `/` in `controller releases/download/...`).
Also, I'd pick the latest stable [`v0.27.1`](https://github.com/actions/actions-runner-controller/releases/tag/v0.27.1).

Reply:
Great catch! (copy pasted from their quickstart :laughing: )

Reply:
Fixed, thank you!